BlueEastCode/bluerain-os
# Fix/debugger typos

## What I did
Fixed typos error in debugger registry.

## How to test
No test case

Is this testable with jest or storyshots?
No
Are any of the existing tests are failing ?
No
Does this need an update to the documentation?
No
If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
